app/components/wallet/file-import/WalletFileImportDialog.js

```diff
@@ -215,7 +215,9 @@ export default class WalletFileImportDialog extends Component {
           <FileUploadWidget
             {...walletFile.bind()}
             selectedFile={walletFile.value}
-            onFileSelected={walletFile.onChange}
+            onFileSelected={(file) => {
+              walletFile.set(file);
+            }}
```

Comment:
Why not keep it simple as before? `onFileSelected={walletFile.set}` 😉

Reply:
Tried that but it does not work... Believe it or not ;)

Reply:
Tried that but it does not work... Believe it or not ;)

Reply:
Ah ok … so probably the method is not bound then 😉  then put a comment there so we know for future reference 👍

Reply:
@DominikGuzei comment added - hope it is fine now ;)